ios/FluentUI/People Picker/PeoplePicker.swift

```diff
@@ -89,7 +89,7 @@ open class PeoplePicker: BadgeField {
     }
 
     /// The `suggestedPersonas` are personas that appear in the persona list that can be picked and added to `pickedPersonas`.
-    private var suggestedPersonas: [Persona] = [] {
+    open var suggestedPersonas: [Persona] = [] {
```

Comment:
>open [](start = 4, length = 4)

please add @objc to public members so they're available to our objC clients as well

Reply:
reverted this change